### PR TITLE
data:B2prakash

### PR DESCRIPTION
**Pull request by B2prakash**
## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## Note to reviewers

you'll are doing  a great job . i would like my Linkfree profile to be updated 

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5674"></a>

### CODE REVIEW

**Review by TRohit20 — DISMISSED** [review]
LGTM

File: data/B2prakash.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      "group": "Primary socials",
+      "name": "Twitter: Follow me",
+      "url": "https://twitter.com/vedprak39212943",
+      "icon": "FaTwitter",
+      "color": "#00ACEE"
+    },
+    {
+      "group": "Primary socials",
+      "name": "linkedin: connect with me ",
+      "url": "https://linkedin.com/in/vedprakash",
```
Comment (kumarsonsoff3) [inline data/B2prakash.json:29]: Not a blocker, but it seems like someone else's profile. If it is not yours, then please update it with yours. 
Rest looks good to me!

image

**Review by iamthecloverly — APPROVED** [review]

File: data/B2prakash.json (hunk 1 above)
Comment (B2prakash) [inline data/B2prakash.json:29]:
```suggestion
      "url": "https://linkedin.com/in/ved prakash",
```

**Comment by SaraJaoude** [comment]
I will merge this Pull Request but your LinkedIn link does not appear to work.  If you wish to make a change please raise a new Pull Request - you may wish to have a look at our Editing Guide when you do: https://linkfree.eddiehub.io/docs/how-to-guides/editing

**Pull request merged**